packages/thirdweb/src/chainsaw/endpoints/getTransactions.ts

```diff
@@ -0,0 +1,109 @@
+import type { ThirdwebClient } from "../../client/client.js";
+import type { Address } from "../../utils/address.js";
+import { getClientFetch } from "../../utils/fetch.js";
+import type { Prettify } from "../../utils/type-utils.js";
+import { formatChainsawTransactions } from "../formatter.js";
+import { addRequestPagination } from "../paging.js";
+import type {
+  ChainsawPagingParams,
+  ChainsawResponse,
+  ChainsawTransactions,
+  Transactions,
+} from "../types.ts";
+import { getTransactionsEndpoint } from "../urls.js";
+
+export type GetTransactionsParams = Prettify<
+  {
+    /**
+     * A client is the entry point to the thirdweb SDK. It is required for all other actions.
+     *
+     * You can create a client using the `createThirdwebClient` function.
+     * Refer to the [Creating a Client](https://portal.thirdweb.com/typescript/v5/client) documentation for more information.
+     *
+     */
+    client: ThirdwebClient;
+    /**
+     * Contract address to fetch transactions for
+     */
+    to: Address;
+    /**
+     * Chain IDs to fetch transactions from. If omitted, will look on all supported chains
+     */
+    chainIds?: number[];
+    /**
+     * Start of the date range to search in
+     */
+    startDate?: Date;
```

Comment:
Could you explain default behavior for these?

Reply:
Added default value comment in [47831d3](https://github.com/thirdweb-dev/js/pull/4004/commits/47831d3ef8dd73c160ae0d4bd1f31687da47b9ca)